app/pandas.ipynb

```diff
@@ -25,17 +25,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 15,
```

Comment:
Cannot see the preview of file because of this error. Add execution count key after the last cell_type
<img width="627" alt="image" src="https://github.com/mate-academy/py-pandas/assets/61380547/e83ee6c4-d7bc-4de3-9111-4938d13da297">